gitlab/integration.sh

```diff
@@ -175,61 +192,68 @@ export CURLOPTS="--fail -sq -m 30 -b $COOKIEJAR"
 CSRFTOKEN=$(curl $CURLOPTS -c $COOKIEJAR "http://localhost/domjudge/login" 2>/dev/null | sed -n 's/.*_csrf_token.*value="\(.*\)".*/\1/p')
 # Make a second request with our session + csrf token to actually log in
 curl $CURLOPTS -c $COOKIEJAR -F "_csrf_token=$CSRFTOKEN" -F "_username=admin" -F "_password=$ADMINPASS" "http://localhost/domjudge/login"
-
-# Send a general clarification to later test if we see the event.
-curl $CURLOPTS -F "sendto=" -F "problem=1-" -F "bodytext=Testing" -F "submit=Send" \
```

Comment:
I think we can still send the clarification here as the domserver is online.

Reply:
Done.